.env.example

```
@@ -0,0 +1,12 @@
+POSTGRES_DB="auth"
+POSTGRES_USER="auth"
+POSTGRES_PASSWORD="auth"
+POSTGRES_PORT="5432"
+POSTGRES_DSN="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:${POSTGRES_PORT}/${POSTGRES_DB}?sslmode=disable"
+
+MIGRATION_DIR="./migrations"
+MIGRATION_DSN="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:${POSTGRES_PORT}/${POSTGRES_DB}?sslmode=disable"
+
+GRPC_PORT="50052"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)